spyder/plugins/variableexplorer/widgets/collectionseditor.py

```diff
@@ -66,6 +67,10 @@
 LARGE_NROWS = 100
 ROWS_TO_LOAD = 50
 
+NUMERIC_NUMPY_TYPES = (int64, int32, int16, int8, uint64, uint32, uint16,
+                       uint8, float64, float32, float16, complex128,
+                       complex64, bool_)

```

Comment:
This constant is defined in `spyder-kernels`, so I think we should import from it instead of redefining it here.

Reply:
I think actually there is no constant in Spyder kernels, instead a variable is defined in the display value function: 

https://github.com/spyder-ide/spyder-kernels/blob/17cac385ad393a51ac5dd26c434b4db455c863eb/spyder_kernels/utils/nsview.py#L343-L354

Should I convert the variable into a constant an then used it in Spyder?

Reply:
I just saw that you did that and I think it was the right call. Thanks @dalthviz!